Avahi support in 0.8.4 series requires at maximum 0.5 version. The current test
does not take this in consideration, so this patch make sure that is not auto
detected with 0.6 version anyway.

Index: vlc-0.8.4/configure.ac
===================================================================
--- vlc-0.8.4.orig/configure.ac
+++ vlc-0.8.4/configure.ac
@@ -4283,7 +4283,7 @@ AC_ARG_ENABLE(bonjour,
   [  --enable-bonjour        Bonjour services discovery (default enabled)])
 if test "${enable_bonjour}" != "no"
 then
-  PKG_CHECK_MODULES(BONJOUR, avahi-client >= 0.3,
+  PKG_CHECK_MODULES(BONJOUR, avahi-client >= 0.3 avahi-client < 0.6,
     [AC_DEFINE(HAVE_AVAHI_CLIENT, [], [Define if you have the avahi-client library])
       VLC_ADD_LDFLAGS([bonjour access_output_http],[$BONJOUR_LIBS])
       VLC_ADD_CFLAGS([bonjour access_output_http],[$BONJOUR_CFLAGS])
